doc/source/examples/dem/granular-dam-break/granular-dam-break.rst

```diff
@@ -0,0 +1,218 @@
+
+==================================
+Granular dam-break
+==================================
+
+This example simulates the dam-break flow of a dry granular material using the discrete element method (DEM). More information regarding the DEM parameters are given in the Lethe documentation, i.e. `DEM parameters <../../../parameters/dem/dem.html>`_.
+
+
+Features
+----------------------------------
+- Solvers: ``dem_3d``
+- Three-dimensional problem
+- Moving floating mesh
+- Load-balancing
+
+
+
+Files used in this example
+----------------------------
+
+- Parameters file for the problem (H=20cm): ``examples/dem/3d-dam-break/granular-dam-break.prm``
+- Parameters file for the H=40cm bed: ``examples/dem/3d-dam-break/granular-dam-break.prm``
```

Comment:
Shouldn't these be two different prm files?

Reply:
Yup, my mistake :)!